2.9/lttng-docs-2.9.txt

```diff
@@ -3823,6 +3823,10 @@ tar -xf lttng-tools-latest-2.9.tar.bz2 &&
 cd lttng-tools-2.9.* &&
 ./configure --libdir=/usr/local/lib32 CFLAGS=-m32 CXXFLAGS=-m32 \
             LDFLAGS='-L/usr/local/lib32 -L/usr/lib32' &&
+	    --disable-bin-lttng \
+	    --disable-bin-lttng-crash \
+	    --disable-bin-lttng-relayd \
+	    --disable-bin-lttng-sessiond &&
```

Comment:
You can use the line up to say 72 columns, so `--disable-bin-lttng` and `--disable-bin-lttng-crash` for example should fit on the same line.

Also use spaces instead of tabs.